assets/prototype/application/ui/components/input/espressoButton/EspressoButton.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { EspressoButtonProps, EspressoButtonSize, EspressoButtonStyle } from './types';
+import { EspressoIcon } from '../../display/espressoIcon';
+
+/**
+ * Button wrapper for adding styles
+ */
+const EspressoButton: React.FunctionComponent<EspressoButtonProps> = ({
+	buttonText,
+	icon,
+	onClick,
+	size,
+	style,
+	htmlClass,
+	...buttonProps
+}) => {
+	let classes = classNames({
+		[htmlClass]: htmlClass,
+		'esprs-button': true,
+		'esprs-btn-accent': style === EspressoButtonStyle.ACCENT,
+		'esprs-btn-default': style === EspressoButtonStyle.DEFAULT,
+		'esprs-btn-primary': style === EspressoButtonStyle.PRIMARY,
+		'esprs-btn-secondary': style === EspressoButtonStyle.SECONDARY,
+		'esprs-btn-tiny': size === EspressoButtonSize.TINY,
+		'esprs-btn-small': size === EspressoButtonSize.SMALL,
+		'esprs-btn-big': size === EspressoButtonSize.BIG,
+		'esprs-btn-huge': size === EspressoButtonSize.HUGE,
+	});
+	let iconSize = 24;
+	switch (size) {
+		case EspressoButtonSize.TINY:
+			iconSize = 18;
+			break;
+		case EspressoButtonSize.SMALL:
+			iconSize = 21;
+			break;
+		case EspressoButtonSize.BIG:
+			iconSize = 27;
+			break;
+		case EspressoButtonSize.HUGE:
+			iconSize = 30;
+			break;
+	}
```

Comment:
I've used enums here as well:
![image](https://user-images.githubusercontent.com/1477453/72290159-466ead80-3655-11ea-8086-bd6fd15b7e53.png)

Reply:
good catch. I missed that. will fix